test/src/test/java/org/corfudb/runtime/checkpoint/CheckpointSmokeTest.java

```diff
@@ -61,13 +61,13 @@ public void testEmptyMapCP() throws Exception {
 
         // Verify that a CP wasn't generated
         long address = mcw.appendCheckpoints(r, "A1").getSequence();
-        assertThat(address).isEqualTo(-1);
+        assertThat(address).isEqualTo(0);
```

Comment:
0 means? If it's possible and presented in `Address` class, please use Adress.SOME_CONSTANT for that

Reply:
0 is not a constant. It is the marker entry placed at 0. I have made it a constant in the test and added comments for clarification.